### PR TITLE
configs/ot2_defconfig: select screen, python3_curses, tmux

### PR DESCRIPTION
**Pull request by mcous**
## overview

This PR branched from / taking over for #90 for CI reasons. Hopefully this branch will preserve @theosanderson's contribution credit!

After some internal discussion, we decided to add both screen and tmux. To quote @sfoster1:

> `tmux` is better than `screen` in 99% of circumstances but the 1% is that you can point `screen` at a serial character device trivially

## changes

- Adds `screen`
- Adds `tmux`
- Adds Python support for the curses module

### CODE REVIEW

**Comment by mcous** [comment]
Looks like I have a little more work to do on this one:

```shell
# tmux
tmux: need UTF-8 locale (LC_CTYPE) but have ANSI_X3.4-1968
```

**Comment by theosanderson** [comment]
Thanks a bunch for taking this forward!

**Comment by mcous** [comment]
@theosanderson thanks for taking the initiative to kick this off! I just detached and reattached to a tmux session on my testing Pi and I'm very excited to have this included

File: configs/ot2_defconfig [hunk 1]
```diff
@@ -8,6 +8,7 @@ BR2_TARGET_GENERIC_HOSTNAME="opentrons"
 BR2_TARGET_GENERIC_ISSUE="Welcome to the Opentrons OT2"
 BR2_INIT_SYSTEMD=y
 # BR2_TARGET_GENERIC_REMOUNT_ROOTFS_RW is not set
+BR2_GENERATE_LOCALE="C.UTF-8 en_US.UTF-8"
```
Comment (mcous) [inline configs/ot2_defconfig:11]: Don't _really_ know if we need to generate en_US given this machine is laboratory equipment that's mainly meant to talk to other equipment / clients. Could keep/remove if anyone felt strongly either way

**Review by sfoster1 — APPROVED** [review]
I can run `screen` and `tmux`, and `import curses` and run some test functions. LGTM

File: configs/ot2_defconfig (hunk 1 above)
Comment (sfoster1) [reply configs/ot2_defconfig:11]: The only reason not to, to me, would be file size, and the builds here don't look too much bigger.

**Pull request merged**